### DISCUSSION TITLE
Custom Field

### DISCUSSION THREAD

**Discussion by gafain** [postfixadmin/postfixadmin#790] [General]
Hello, 
I need a custom Field in the mailbox table for our use.
In the old version of postfixadmin I was able to add this Field.
In this version is more "complicated". I Think I need to edit the MailboxHandler.php but I'm not sure.
Can you give me some help ?

Best regards

**Comment by bnchdan** [comment]
Hi,
If you are still interested. What exactly do you want to add?

**Comment by DavidGoodwin** [comment]
"we" probably need some documentation with a token example covering this - e.g. add a column to the database, find the appropriate Handler class and add a pacol definition in etc etc
